app/controllers/user.controller.js

```diff
@@ -10,19 +10,31 @@ exports.create = async (req, res) => {
   if (req.body.firstName === undefined) {
     const error = new Error("First name cannot be empty for user!");
     error.statusCode = 400;
-    throw error;
+    res.status(400).send({
```

Comment:
in addition to returning the error information in a HTTP 400 response, consider also logging the error with `console.log` so that the backend service has a recorded record of the problem